base/docs/helpdb/Base.jl

```diff
@@ -6225,7 +6229,10 @@ nextind
 """
     >>>(x, n)
 
-Unsigned right bit shift operator.
+Unsigned right bit shift operator, `x >>> n`. The result is `x` shifted right by `n` bits,
+where `n >= 0`, filling with `0`s.
```

Comment:
Another way of describing this is that `x` is first reinterpreted as unsigned, then shifted via `>>`, then reinterpeted back as the original type.